### DISCUSSION TITLE
How to access an account which has two factor authentication enabled and you forgot both the authentication key and the github password?

### DISCUSSION THREAD

**Discussion by jafarfahimi** [community/community#114375] [Other Feature Feedback, Questions, & Ideas]
### Forgotten Github Password and 1Password Authentication Keys - Seeking Account Recovery and Deletion Options

Question

### Body

Hello everyone,

I am facing an issue with my Github account that has two-factor authentication enabled through 1Password. Unfortunately, I have forgotten both my Github password and the authentication keys for 1Password. When I try to reset my password on Github, the reset email is sent to my Gmail account. I have reached out to the 1Password team for assistance, but they are unable to help me recover my account without the necessary password and authentication keys.

Even if I were to unlink my Github account from 1Password, I am still prompted to enter a two-factor authentication key when logging in to Github. 

I am seeking advice on how to regain access to my account in this situation where I have forgotten both the authentication key and the Github password. Additionally, if there is a solution for deleting my account, which I am unable to do normally, please provide guidance on that as well. Any help or suggestions would be greatly appreciated. 
[redacted]
Thank you.

**Comment by pulkitxm** [comment]
You can follow the instructions outlined in GitHub's documentation for recovering your account if you've lost your two-factor authentication credentials. Here's a summary of the steps:

Visit the [Account Recovery](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) Page: Go to GitHub's account recovery page.

Provide Necessary Information: You'll need to provide your username, email address, and a few other details to verify your identity.

Contact Support: Once you've provided the required information, GitHub Support will review your request and assist you with regaining access to your account.

Regarding account deletion, if you're unable to delete your account through the usual means due to the 2FA issue, you can reach out to GitHub Support for assistance with deleting your account.

Feel free to contact GitHub Support for further guidance and assistance with both account recovery and deletion.

**Reply by jafarfahimi** [reply]
Thanks dear Pulkitxm, 
I read the "Account Recovery" page info. According to that page, I can recover my account in case I have any of "Verified device, SSH key or personal access token". Unfortunately, I don't have any of them so the solution, as mentioned in the page, is to unlink my email and let go of my Github account.
Well, this answer is hard to accept for me and every one who has worked on his Github account for years. I just wanted to see if there's another solution. I guess I should contact Github Support team.

** I'm stuck in second step of recovery:
![step 2](https://github.com/community/community/assets/163999985/2898f0b4-4836-402f-813b-b9831c07855a)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @jafar-fahimi ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

Note: I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).
